core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -458,33 +485,39 @@ default Builder retryPolicy(RetryMode retryMode) {
         RetryPolicy retryPolicy();
 
         /**
-         * Configure the retry mode used to determine the retry strategy that is used when handling failure cases. This is
-         * shorthand for {@code retryStrategy(SdkDefaultRetryStrategy.forRetryMode(retryMode))}, and overrides any configured
-         * retry policy on this builder.
+         * Configure the retry mode used to resolve the corresponding {@link RetryStrategy} that should be used when handling
+         * failure cases.
+         *
+         * @see RetryMode
          */
-        default Builder retryStrategy(RetryMode retryMode) {
-            return retryStrategy(SdkDefaultRetryStrategy.forRetryMode(retryMode));
-        }
+        Builder retryStrategy(RetryMode retryMode);
```

Comment:
Seems like breaking change, same for other methods which we took out default implementation from

Reply:
It's marked as breaking by japicmp but I don't know what else we can do about it. Any suggestions?

Reply:
Can we keep the original implementation? Or alternatively, we can change it to throw UnsupportedOperationException

Reply:
Oh, that's a good idea, keep it default and throw and override it in the impl, yeah, we can do that, let me update the PR.